cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -503,25 +503,33 @@ impl PathEnvironment {
         replaced = replaced.replace(FLOX_INSTALL_PLACEHOLDER, packages);
 
         // Replace profile
-        let profile = if let Some(ref custom_profile) = customization.profile {
+        let profile = formatdoc! {r#"
```

Comment:
Again let's pass this to product to decide. Speaking with Ross he really liked this specific example because it highlights one fairly well-known thing (setting "vi mode" for your cmdline) using different syntax for different shells.